### DISCUSSION TITLE
Change page background on each transition

### DISCUSSION THREAD

**Discussion by luiscabre** [swup/swup#344] [Help] [answered]
I'm using Swup ..and it working fine so far ...I have 4 pages ( home, about, work, contact ) .....I made a transition fade and it works fine ...but now I'd like to add different ( colors or backgound image ) on each page.  I tried to put a class  on each page's body, but it doesn't works  ... Please I need help with this.
 
![about](https://user-images.githubusercontent.com/17053257/110136539-30b2cf00-7da6-11eb-8e06-47cd00cbd9c7.png)
![documentation](https://user-images.githubusercontent.com/17053257/110136544-314b6580-7da6-11eb-9ba2-ca5b0ecf8182.png)
![homepage](https://user-images.githubusercontent.com/17053257/110136545-314b6580-7da6-11eb-82ae-6e0253d7403f.png)

**Comment by gmrchk** [comment, marked as answer]
I see your styles are included for each page in the head tag. You can use [Head plugin](https://swup.js.org/plugins/head-plugin) to make swup replace the tags in `head`. However, I would rather recommend putting all styles (at least the ones required for general layout and swup transitions) into one CSS file included on all pages.
